### DISCUSSION TITLE
Istio support

### DISCUSSION THREAD

**Discussion by lordigon** [arkmq-org/arkmq-org-broker-operator#493] [Q&A]
Is there any documentation/example on how to use Istio with Artemis? Here's my use case: I want to use ssl transport for external clients and for cluster internal ones use mqtt transport but with istio. Is it possible to configure ArtemisCluster CR for such use case?

**Comment by brusdev** [comment]
@lordigon I'm not aware of any example but I see other discussions related to Istio, see https://github.com/artemiscloud/activemq-artemis-operator/discussions?discussions_q=Istio
